src/components/player/control-bar-divider.tsx

```diff
@@ -4,6 +4,6 @@ const ControlBarDivider: React.FC<{
   key: string
   order: number
   className: string
-}> = (props) => <div {...props} />
+}> = ({className}) => <div className={className} />
```

Comment:
for divider we don't need to push all props to div and seemed like only `className` was needed. This made a lot of warning disappear in console.